.helm/.env-example

```diff
@@ -8,3 +8,19 @@ POSTGRES_ADMIN_URL=
 BASIC_AUTH_ENABLED=false
 BASIC_AUTH_USERNAME=test
 BASIC_AUTH_PASSWORD=test
+
+BACKUP_SCHEDULE=@hourly
+BACKUP_S3_ENDPOINT=
+BACKUP_S3_BUCKET=
+BACKUP_S3_ACCESS_KEY_ID=
+BACKUP_S3_ACCESS_KEY=
+BACKUP_ENCRYPTION_KEY=test
+
+#RESTORE_SOURCE_FILE=latest
+RESTORE_SOURCE_APP=${instance_name}-1
```

Comment:
Can you explain why you chose this value?

Reply:
the deploy script also allows to deploy the helm chart multiple times in one go.
That's why the deployment app name is in my case bacluc-1.
But we could also remove the value here, because then it uses the app name generated in the chart